go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/minio/minio
 
-go 1.21
+go 1.22.5
```

Comment:
```suggestion
go 1.22
```

Reply:
This change comes from Console it self, Fixing there first